### PR TITLE
[CollectionCells] Add accessibility hint in edit mode

### PR DESCRIPTION
**Pull request by Arcank**
When editing, selectable/deselectable cells now give an accessibility
hint: "Double tap to select.". This helps users to know what they can do
while in edit mode.

Related: #1257

### CODE REVIEW

**Review by randallli — APPROVED** [review]

File: components/CollectionCells/src/MDCCollectionViewCell.m [hunk 1]
```diff
@@ -478,8 +492,25 @@ - (UIAccessibilityTraits)accessibilityTraits {
   return accessibilityTraits;
 }
 
+- (NSString *)accessibilityHint {
```
Comment (randallli) [inline components/CollectionCells/src/MDCCollectionViewCell.m:493]: can we and a unit test?

**Review by randallli — CHANGES_REQUESTED** [review]

File: components/CollectionCells/src/MDCCollectionViewCell.m (hunk 1 above)
Comment (Arcank) [reply components/CollectionCells/src/MDCCollectionViewCell.m:493]: Added tests.

**Comment by Arcank** [comment]
Kindly ping.

**Comment by Arcank** [comment]
@randallli: Can you have a look? I added test coverage.

**Comment by randallli** [comment]
Sorry will look when I get in.

**Review by randallli — APPROVED** [review]

**Pull request merged**